src/data/setoid.lean

```diff
@@ -0,0 +1,603 @@
+/-
+Copyright (c) 2019 Amelia Livingston. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Amelia Livingston, Bryan Gin-ge Chen
+-/
+
+import data.quot data.set.lattice data.fintype order.order_iso order.galois_connection
+
+/-!
+# Equivalence relations
+
+The first section of the file defines the complete lattice of equivalence relations 
+on a type, results about the inductively defined equivalence closure of a binary relation, 
+and the analogues of some isomorphism theorems for quotients of arbitrary types.
+
+The second section comprises properties of equivalence relations viewed as partitions, and 
+a function "bell" that computes Bell numbers - the numbers of possible partitions of a finite 
```

Comment:
```suggestion
a function `bell` that computes Bell numbers - the numbers of possible partitions of a finite 
```
I think this syntax for quoting names in the code has better semantics. Also elsewhere. Feel free to disagree.